src/data/markdown/docs/30 xk6-browser/01 xk6-browser/01 Browser.md

```diff
@@ -7,18 +7,18 @@ excerpt: "xk6-browser: Browser Class"
-| [browser.close()](/javascript-api/xk6-browser/browser/close)                                   | Closes the browser and all of its pages (if any were opened).                                                   |
-| [browser.contexts()](/javascript-api/xk6-browser/browser/contexts)                             | Allows you to access all open browser contexts.                                                                 |
-| [browser.isConnected](/javascript-api/xk6-browser/browser/isconnected)                         | Indicates whether the WebSocket connection to the browser application is active or not.                         |
-| 🚧 [browser.newBrowserCDPSession()](/javascript-api/xk6-browser/browser/newbrowsercdpsession) | Allows you to access Browser's CDP session and talk directly to the browser application using the CDP protocol. |
-| [browser.newContext([options])](/javascript-api/xk6-browser/browser/newcontext/)               | Creates and returns a new browser context.                                                                      |
-| [browser.newPage([options])](/javascript-api/xk6-browser/browser/newpage)                      | Creates a new Page in a new BrowserContext and returns the page.                                                |
-| 🚧 [browser.on('disconnected')](/javascript-api/xk6-browser/browser/on)                       | Detects events from the browser application.                                                                    |
-| ❌ [browser.startTracing()](/javascript-api/xk6-browser/browser/starttracing)                  | Starts Chromium Tracing for debugging a Page.                                                                   |
-| ❌ [browser.stopTracing()](/javascript-api/xk6-browser/browser/stoptracing)                    | Stops the tracing created by the browser.startTracing() method.                                                 |
-| [browser.version()](/javascript-api/xk6-browser/browser/version)                               | Returns the browser application's version.                                                                      |
+| State | Method                                                                                         | Description                                                                                                     |
+| - | ---------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------- |
+| ✅ | [browser.close()](/javascript-api/xk6-browser/browser/close)                                   | Closes the browser and all of its pages (if any were opened).                                                   |
```

Comment:
Yeah, good point. I was wondering how it would look if the correct UTF8 fonts weren't available. Is this something I can request now or at least start the conversation going?

Reply:
Created an issue to track this: https://github.com/grafana/k6-docs/issues/729